.travis.yml

```diff
@@ -1,11 +1,37 @@
-language: python
+language: node_js
 python:
-  - "2.7"
-before_script:
-  - psql -c 'create database oneanddone;' -U postgres
-  - cp oneanddone/settings/local.py-travis oneanddone/settings/local.py
+- "2.7"
+node_js:
+- "0.10"
+addons:
+  artifacts:
+    paths:
+    - results
+    permissions: public-read
+    bucket: fireplace-tests
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
 install:
-  - pip install -r requirements.txt
+  - "sudo pip install -r requirements.txt"
+  - "sudo pip install -r oneanddone/functional_tests/requirements.txt"
```

Comment:
I would suggest a `requirements` directory with `dev.txt` and `test.txt` or similar.